scripts/snakemake_adage_two_level.R

```diff
@@ -0,0 +1,202 @@
+library(ADAGEpath)
+library(DT)
+library(readr)
+library(tidyr)
+library(plyr)
+
+# This script uses ADAGEpath functions to do comparisons between sets of sputum
+# samples and controls to determine differentially activated signatures
+
+# determine control set from snakemake wildcards -------------------------
+
+strain <- snakemake@wildcards[['strain']]
+# strain <- "pao1"
```

Comment:
You probably don't need these commented bits anymore

Reply:
This always gets pointed out. I like to leave them in in case I need to return to the script and try things be hand at any point. it's to hard to re-figure out what goes where if I don't keep them